.gitignore

```diff
@@ -34,3 +34,7 @@ _work
 
 *.tgz
 charts/operator/crds
```

Comment:
these seem to be some leftovers from our helm work. while we're at it, maybe drop?

Reply:
sure, removed